src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -707,6 +708,25 @@ private void ParseHelper(string[] args)
                         break;
                     }
                 }
+
+                else if (MatchSwitch(switchKey, "settingsfile", "settings") )
+                {
+                    ++i;
+                    if (i >= args.Length)
+                    {
+                        WriteCommandLineError(
+                            CommandLineParameterParserStrings.MissingSettingsFileArgument);
+                        break;
+                    }
+                    string configFile = args[i];
+                    if (!System.IO.File.Exists(configFile))
+                    {
+                        WriteCommandLineError(
+                            CommandLineParameterParserStrings.SettingsFileNotExists);
```

Comment:
The error message should also be similar to what's used for `-File`.
```
PS:3> powershell -File C:\Users\nonexist.ps1
The argument 'C:\Users\nonexist.ps1' to the -File parameter does not exist. Provide the path to an existing '.ps1' file as an argument to the -File parameter.
```

Reply:
Will do